pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "vanilla-installer"
-version = "v1.0.0-dev6"
+version = "v1.0.0-dev7"
 description = "An installer for Fabulously Optimized for the vanilla launcher."
```

Comment:
The tone doesn't seem to make sense to me, Shouldn't it be `An installer of Fabulously Optimized for the vanilla launcher.` ?

Reply:
Yeah that seems better, I'm not too good at these lol